src/plugins/foxPage/components/FoxTab.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box } from '@chakra-ui/layout'
+import { SkeletonText, Tab, useColorModeValue } from '@chakra-ui/react'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
+import { Card } from 'components/Card/Card'
+
+type FoxTabProps = {
+  assetIcon: string
+  assetSymbol: string
+  fiatAmount: string
+  cryptoAmount: string
+  isSelected?: boolean
+}
+
+export const FoxTab = ({
+  assetIcon,
+  assetSymbol,
+  fiatAmount,
+  cryptoAmount,
+  isSelected,
+}: FoxTabProps) => {
+  const bgHover = useColorModeValue('gray.100', 'gray.750')
+
+  return (
+    <Tab
+      _selected={{ bg: bgHover, borderColor: 'primary', borderWidth: '2px' }}
```

Comment:
Maybe a storybook side effect, but I have some kind of double selection (border) when clicking on a tab on my side.

See the difference between clicked and clicked outside
![foxtab](https://user-images.githubusercontent.com/5720927/168600896-7aaad209-c379-4d35-9334-9c7dd2a09ef2.png)

![foxtab2](https://user-images.githubusercontent.com/5720927/168600916-194ffcc2-809d-459e-84c7-b2ededf6d9c7.png)

Reply:
This is not a side effect, it's the active and focus state (i18n thing of Chakra components), we probably want to keep it like that in order to let people use our app navigate with the keyboard

Reply:
Interesting, thank you for this answer. It seems to work well, it's just I haven't seen this behaviour in the web app currently, hence I am wondering what kind of behaviour is really wanted (this question may be asked to product in the upcoming meeting)

Reply:
Maybe we shouldn't add the border to the child element in case it's active, that could be a fix

Reply:
Fixed it, should be fine now!

Reply:
Perfect !